packages/core/test/metadata-delegate.test.js

```diff
@@ -0,0 +1,37 @@
+import { add, clear } from '../lib/metadata-delegate'
+
+// it doesn't seem easy or even impossible to check whether __proto__ keys can be overwritten
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/proto
+// so tests are only for prototype and constructor
+
+describe('metadata delegate', () => {
+  describe('clear', () => {
+    it('should not overwrite prototype keys', () => {
```

Comment:
You could also use [.each](https://jestjs.io/docs/api#testeach) in this kind of circumstance, but just a suggestion!